components/widgets/menu/menu_items/menu_item.scss

```diff
@@ -1,5 +1,8 @@
 @import '../../../../sass/utils/module';
 
+$menu_item_warn: #ffd47029;
+$menu_item_critical: #d24b4e29;
```

Comment:
why add these variable colors here? if those are new colors from the design, shouldn't the theme templates colors be updated?